master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring IP-in-IP
+---
+
+Calico can be configured to use IP-in-IP encapsulation for inter-host inter-workload
```

Comment:
I wonder if we want to go deeper into the "why" here. 

- Why you'd use IP-IP encapsulation ?
  - when networking fabric does RPF and drops unrecognized traffic
- Why you'd use cross-subnet? 
  - when hosts are on a L2 hop and thus only need IPIP for L3 hop traffic